File: OpenCV/13_esquinas/cpp/main.cpp
#include <opencv2/opencv.hpp>
#include <iostream>
int main(){ auto g=cv::imread("input.jpg",cv::IMREAD_GRAYSCALE); if(g.empty()) return 1; std::vector<cv::Point2f> c; cv::goodFeaturesToTrack(g,c,300,0.01,8,cv::noArray(),3,true,0.04); std::cout<<"esquinas "<<c.size()<<"\n"; }

File: OpenCV/13_esquinas/python/main.py
import cv2
g=cv2.imread("input.jpg",cv2.IMREAD_GRAYSCALE); assert g is not None
corners=cv2.goodFeaturesToTrack(g,300,0.01,8,useHarrisDetector=True,k=0.04); print("esquinas",0 if corners is None else len(corners))
